classes/range.js

```diff
@@ -1,3 +1,5 @@
+const SPACE_CHARACTERS = /\s+/g
```

Comment:
This is only used once, there's no need to abstract it to a variable.

Reply:
FYI: This is a performance optimization, not a stylistic choice. Reconstructing a new `Regex` instance each time the `Range` class is instantiated is much more expensive then reusing the same one.